src/CSharpExtDownloader.ts

```diff
@@ -54,9 +54,10 @@ export class CSharpExtDownloader {
     }
 }
 
-export function GetRunTimeDependenciesPackages(packageJSON: any): Package[] {
+export function GetRunTimeDependenciesPackages(packageJSON: any, extensionPath: string): PackageJSONPackage[] {
     if (packageJSON.runtimeDependencies) {
-        return JSON.parse(JSON.stringify(<Package[]>packageJSON.runtimeDependencies));
+        let dependencies = <PackageJSONPackage[]>packageJSON.runtimeDependencies;
+        return dependencies.map(pkg => new Package(pkg, extensionPath));
```

Comment:
I don't get it. This method returns an array of `Package`s. So why is the return type an array of `PackageJSONPackage`?

Reply:
Yes you are right. Will correct that.